homeassistant/components/fortios/device_tracker.py

```diff
@@ -46,29 +46,63 @@ def get_scanner(hass, config):
         _LOGGER.error("Failed to login to FortiOS API: %s", ex)
         return None
 
-    return FortiOSDeviceScanner(fgt)
+    status_json = fgt.monitor("system/status", "")
+    fos_major_version = int(status_json["version"][1])
```

Comment:
Will the version string always be convertable to an integer?

Reply:
short answer is yes!